packages/components/src/Text/Text.tsx

```diff
@@ -1,12 +1,17 @@
 import React, { PropsWithChildren } from "react";
 import { Typography, TypographyOptions, TypographyProps } from "../Typography";
 
+type TextElement = Extract<
+  TypographyProps["element"],
+  "p" | "b" | "em" | "dd" | "dt" | "strong" | "span"
```

Comment:
Semantically I bet you could probably end up with a Text that's somehow an h4/5/6 🤔

Reply:
This was my thoughts behind it https://github.com/GetJobber/atlantis/pull/2515#discussion_r2060498140
We can always add it if need be but I personally think we can do this and expand as needed. Basically headings looking like Text seems odd